scripts/Muon/GUI/ElementalAnalysis/Plotting/plotting_presenter.py

```diff
@@ -48,3 +59,89 @@ def add_moveable_vline(self, plot_name, x_value, y_minx, y_max, **kwargs):
 
     def add_moveable_hline(self, plot_name, y_value, x_min, x_max, **kwargs):
         pass
+
+    def removeSubplotConnection(self, slot):
+        self.view.subplotRemovedSignal.connect(slot)
+
+    def close(self):
+        if self.selectorWindow is not None:
+            self.closeSelectorWindow()
+        if self.rmWindow is not None:
+            self.closeRmWindow()
+
+    def add(self):
```

Comment:
For a TODO functionality, I would consider either hiding the button in the interface or disabling it with a tooltip text explaining that it is still 'under construction'.

Reply:
I can not add it to the toolbar for now.